table.csv

```diff
@@ -143,6 +143,7 @@ car-index-sorted,               serialization,  0x0400,         draft,      CARv
 car-multihash-index-sorted,     serialization,  0x0401,         draft,      CARv2 MultihashIndexSorted index format
 transport-bitswap,              transport,      0x0900,         draft,      Bitswap datatransfer
 transport-graphsync-filecoinv1, transport,      0x0910,         draft,      Filecoin graphsync datatransfer
+transport-ipfs-gateway-http,    transport,      0x0920,         draft,      HTTP IPFS Gateway trustless datatransfer
```

Comment:
:thought_balloon:  This terribly long string will be sent with every IPNI roundtrip, and due to the way IPNI implemented /routing/v1 API, we use string that can change at atny time, and not the number: https://github.com/ipfs/specs/issues/377

I'd be strongly recommending moving to number instead of string, which we did change multiple times in the past ( `/ipfs/` → `/p2p`, then `/ipns-ns` → `/ipns` etc)

Reply:
ℹ️ the cid.contact find response https://github.com/ipni/specs/blob/main/schemas/v1/openapi.yaml#L120 does use compact number versions

Reply:
IPNI [metadata](https://github.com/ipni/specs/blob/main/IPNI.md#metadata) always uses the varint numerical value.

Reply:
sgtm, I've opened https://github.com/ipfs/specs/pull/403 for explicitly supporting hex codes on `/routing/v1`